CMakeLists.txt

```diff
@@ -2745,6 +2745,11 @@ install(EXPORT netCDFTargets
   NAMESPACE netCDF::
   )
 
+export(EXPORT netCDFTargets
+  FILE "${CMAKE_CURRENT_BINARY_DIR}/netCDFTargets.cmake"
```

Comment:
`${CMAKE_CURRENT_BINARY_DIR}` is unnecessary if you want to simplify it like you did with `write_basic_package_version_file`